tuneme/templates/polls/poll_page.html

```diff
@@ -5,7 +5,7 @@
 		<h2>{% trans "What's your opinion?" %}</h2>
 		<div class="block turq">
 			<div class="title">
-				<div class="icon"><img src="{% static 'img/pie.png' %}" alt="Have Your Say"></div>
+				<div class="icon"><a href=""><img src="{% static 'img/pie.png' %}" alt="Have Your Say"></a></div>
```

Comment:
@milton in order to go to the polls results page the URL needs a vote value, is there anyway that we can go to the results page without voting ?